source/common/ssl/openssl.h

```diff
@@ -4,25 +4,21 @@
 
 #include "openssl/ssl.h"
 
+#ifndef OPENSSL_IS_BORINGSSL
+#error Envoy requires BoringSSL
+#endif
+
 namespace Ssl {
 
 /**
- * Global functionality specific to OpenSSL.
+ * Global functionality specific to BoringSSL.
  */
 class OpenSsl final {
```

Comment:
nit: I think you mentioned this elsewhere but rename to BoringSsl or just something general like "Global" ?